Documentation/HowToUseETW.md

```diff
@@ -0,0 +1,46 @@
+# Supported
+ETW logs are available on CoreCLR only currently.
+
+
+# How to Collect ETW logs use logman
+
+
+## Start the logging
+
+ * Option 1: Generate a single file that does circular logging with Verbose level, For example
+ 
+ ```
+logman start WCFETWTracing -p "Microsoft-Windows-Application Server-Applications" 0xFFFFFFFF  0x5 -bs 64 -nb 120 320 -ets -ct perf -f bincirc -max 500 -o c:\temp\WCFEtwTrace.etl
```

Comment:
This document appears only in OSS, therefore it is only OSS people who will read it.  The right document to reach true end users would be a blog (which is something our team should ultimately consider seriously, along with samples).  It raises a good point -- who is the audience these OSS documentation files speak to?  All the other MD's talk directly to the OSS user (how to build, how to test, what the Bridge is), so I think the purpose of the Documentation folder is to speak to OSS folks.

Therefore I think it is worth at least pointing out any special considerations needed if they are using ETW to look at the behavior of the OuterLoop tests vs. debugging their own application.  A URL to using ETW for WCF on the full framework would also be beneficial.

This doesn't mean a big restructure -- just a few extra paragraphs to help them understand how they can use ETW for the full E2E debugging experience.